sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -327,7 +327,7 @@ namespace Azure { namespace Core { namespace Http {
      * @brief A value indicating whether the returned raw response for this request will be buffered
      * within a memory buffer or if it will be returned as a body stream instead.
      */
-    bool ShouldBufferResponse() { return this->m_shouldBufferResponse; }
+    bool ShouldBufferResponse() const { return this->m_shouldBufferResponse; }
```

Comment:
To confirm, **removing** const from a method is considered  a breaking change, correct?

But adding is fine. Because a const method can be called on either a const or non-const instance, whereas a non-const method cannot be called by a const instance.

In general, I wonder how pessimistic we should be with adding const, to avoid a breaking change, if the implementation were required to change.